libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -95,15 +105,31 @@ public MicrosoftAppCredentials(string appId, string password, HttpClient customH
         /// </value>
         public string MicrosoftAppPassword { get; set; }
 
+        /// <summary>
+        /// Gets or sets the channel auth token tenant for this credential.
+        /// </summary>
+        private string _channelAuthTenant;
+        public string ChannelAuthTenant
+        {
+            get { return string.IsNullOrEmpty(_channelAuthTenant) ? "botframework.com" : _channelAuthTenant; }
```

Comment:
Please move "botframework.com" to the constants file.

Reply:
yep